mscore/jackweakapi.cpp

```diff
@@ -74,16 +73,27 @@ static int tryload_libjack()
     if (getenv("SKIP_LIBJACK") == 0) { // just in case libjack is causing troubles..
     #ifdef __APPLE__
         libjack_handle = dlopen("libjack.0.dylib", RTLD_LAZY);
+        if (!libjack_handle) {
+            qDebug("dlopen error : %s ", dlerror());
+        }
+        libjack_handle = dlopen("/usr/local/lib/libjack.0.dylib", RTLD_LAZY);
+        if (!libjack_handle) {
+            qDebug("dlopen error : %s ", dlerror());
+        }
     #elif defined(WIN32)
         // Force char implementation of library instead of possibly wchar_t implementation to be called.
-        libjack_handle = LoadLibraryA("libjack.dll");
+        #ifdef _WIN64
+            libjack_handle = LoadLibraryA("libjack64.dll");
+        #else
+            libjack_handle = LoadLibraryA("libjack.dll");
+        #endif
```

Comment:
So basically this is the change that resolves the issue on Windows? Other changes seem to be mostly about functions which are commented out anyway.

Reply:
Yes

Reply:
Well, OK, that would be curious if the issue has been just in the library name. Let's merge it then and see if it helps for others.